services/src/unit-test/java/uk/gov/moj/sdt/services/SubmitQueryServiceTest.java

```diff
@@ -484,12 +488,60 @@ void testSubmitQueryCmcNoCmcClaimDefences(SubmitQueryResponse cmcSubmitQueryResp
         verify(mockResponsesSummaryUtil, never()).getSummaryResults(any(), any());
     }
 
+    @ParameterizedTest
+    @NullSource
+    @MethodSource("noCmcClaimDefences")
+    void testSubmitQueryMcolMaximumNumberOfDefencesReached(SubmitQueryResponse cmcSubmitQueryResponse) {
+        // Set up query request
+        ISubmitQueryRequest submitQueryRequest = new SubmitQueryRequest();
+
+        IBulkCustomer requestBulkCustomer = new BulkCustomer();
+        requestBulkCustomer.setSdtCustomerId(12L);
+        submitQueryRequest.setBulkCustomer(requestBulkCustomer);
+
+        ITargetApplication requestTargetApplication = new TargetApplication();
+        requestTargetApplication.setTargetApplicationCode("MCOL");
+        submitQueryRequest.setTargetApplication(requestTargetApplication);
+        submitQueryRequest.setResultCount(2000);
+        IErrorLog errorLog = new ErrorLog();
+        errorLog.setErrorCode("78");
+        errorLog.setErrorText("maximum number of defences reached");
```

Comment:
The error text for code 78 will be "Results truncated - maximum number allowed reached."

Reply:
addressed